src/plugins/grpc/client.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+const { FORMAT_HTTP_HEADERS, Tags } = require('opentracing')
+
+function patch (grpc, tracer, config) {
+  if (config.client === false) return
+
+  config = config.client || config
+
+  function gRPCInterceptor (options, nextCall) {
+    const { path, responseStream } = options.method_definition
+    const scope = tracer.scopeManager().active()
+    const span = tracer.startSpan('grpc.client', {
+      childOf: scope && scope.span(),
+      tags: {
+        [Tags.SPAN_KIND]: Tags.SPAN_KIND_RPC_CLIENT,
+        'grpc.method_name': path,
+        'grpc.method_type': responseStream ? 'server_stream' : 'unary',
+        'resource.name': path.split('/').slice(-1),
```

Comment:
Personal preference but totally willing to change if you prefer it. My thinking was:

* `grpc.method.name` would be something like: `/FooBarService/GetFooBar`
* `resource.name` would be: `GetFooBar`

If we keep the entire path we might as well remove the `grpc.method.name` tag since it would be duplicated.

See screenshot below to see how it looks (in this case I have a "GetUser" resource name):

<img width="1008" alt="dd-trace grpc" src="https://user-images.githubusercontent.com/92085/57109667-fe5d6400-6d2d-11e9-8d47-d3cba9782747.png">

Reply:
It's fine to keep the tag as well. It's possible that we change the value of `resource.name` eventually for example but the method name will always be the same.

My main concern with removing the service name from the resource is that it could cause stats to not be calculated correctly if multiple services have the same method. Right now it wouldn't cause issues but we are planning to add more calculations soon that could be impacted.

Reply:
> My main concern with removing the service name from the resource is that it could cause stats to not be calculated correctly if multiple services have the same method

Would that not apply to the graphql and koa plugins as seen on the screenshot? (`me` and `POST`, respectively)

Reply:
For the server possibly since then you know for sure what the service is. But for a remote endpoint you don't have this information.

Reply:
@nunofgs Do you think the proposed change makes sense?

Reply:
Whoa, totally missed your comment. Just to clarify, you're requesting:

```js
        'grpc.method_name': path,
        'resource.name': path
```

Correct?

Reply:
Correct.

Reply:
PR Updated but fwiw I still don't see much value in sending both tags with the same content. Will defer to your judgement here :)

Reply:
`resource.name` is not really a tag once it's been formatted by the tracer and becomes a field that has a special meaning in our backend and UI. This means it could be altered later in the pipeline for example. It also means its value will be removed from the list of tags. Having a separate tag with the same value ensures the value is still available as an untouched tag.

It's also possible that we eventually stop relying on the `resource` field and switch to use the tag directly instead. In that case, having the tag already available will reduce the overhead of the migration.